src/common/write_csv.py

```diff
@@ -0,0 +1,19 @@
+from common.data_cleaning import remove_special_characters
+from datetime import datetime
+
+def write_post_data(writer, submission):
```

Comment:
I would suggest you to make an habit of adding `type hints` and `return type`. This way, whoever is going the code would understand what the method signature is.